### PR TITLE
[IPCT1-650] - list manager when is a pending community

### PR DESCRIPTION
**Pull request by dev-jotape**
This PR fixes [IPCT1-650] at https://impactmarket.atlassian.net/browse/IPCT1-650

## Changes
on `getManagers` function, return the manager when is a pending community

## Tests
new test created

### CODE REVIEW

File: packages/core/src/services/ubi/community.ts [hunk 1]
```diff
@@ -867,56 +881,85 @@ export default class CommunityService {
...
+            });
+            return [{
+                user: user as AppUser,
+                isDeleted: false,
+                addedBeneficiaries: 0,
+                pending: true,
+            }];
+        } else {
+            const result = await this.manager.findAll({
+                include: [
```
Comment (obernardovieira) [inline packages/core/src/services/ubi/community.ts:912]:
```suggestion
            const result = await this.manager.findAll({
                attributes: ['address'],
                include: [
```

File: packages/core/src/services/ubi/community.ts [hunk 2]
```diff
@@ -867,56 +881,85 @@ export default class CommunityService {
...
+                    {
+                        model: this.appUser,
+                        as: 'user',
+                        include: [
```
Comment (obernardovieira) [inline packages/core/src/services/ubi/community.ts:916]:
```suggestion
                        as: 'user',
                        attributes: ['username'],
                        include: [
```

File: packages/core/src/services/ubi/community.ts [hunk 3]
```diff
@@ -855,7 +856,20 @@ export default class CommunityService {
...
+        addedBeneficiaries: number;
+        pending: boolean;
+        id?: number;
+        address?: string;
+        communityId?: number;
+        active?: boolean;
+        readRules?: boolean;
+        blocked?: boolean;
+        createdAt?: Date;
+        updatedAt?: Date;
```
[the remark this answers is not in the document]
Comment (obernardovieira) [reply packages/core/src/services/ubi/community.ts:870]: well, the address

File: packages/core/src/services/ubi/community.ts [hunk 4]
```diff
@@ -867,56 +881,85 @@ export default class CommunityService {
...
+                ],
+                where: {
+                    address: community.requestByAddress
+                }
+            });
+            return [{
+                user: user as AppUser,
+                isDeleted: false,
+                addedBeneficiaries: 0,
+                pending: true,
```
Comment (obernardovieira) [inline packages/core/src/services/ubi/community.ts:908]: not necessary

File: packages/core/src/services/ubi/community.ts [hunk 5]
```diff
@@ -867,56 +881,85 @@ export default class CommunityService {
...
-        });
+            return result.map((r) => {
+                const manager = r.toJSON() as ManagerAttributes;
+                const addedBeneficiaries = beneficiariesAdded.find(
+                    (el) => el.from === manager.address
+                );
+                return {
+                    ...manager,
+                    isDeleted: !manager.user,
+                    addedBeneficiaries:
```
Comment (obernardovieira) [inline packages/core/src/services/ubi/community.ts:957]: can you rename to just "added", so it goes aligned with the subgraph https://github.com/impactMarket/subgraph/blob/main/schema.graphql#L60
you can rename it, I think these values aren't being used yet.

File: packages/core/src/services/ubi/community.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (dev-jotape) [reply packages/core/src/services/ubi/community.ts]: done

File: packages/core/src/services/ubi/community.ts (hunk 4 above)
Comment (dev-jotape) [reply packages/core/src/services/ubi/community.ts]: done

File: packages/core/src/services/ubi/community.ts (hunk 5 above)
Comment (dev-jotape) [reply packages/core/src/services/ubi/community.ts]: done

File: packages/core/src/database/models/ubi/manager.ts [hunk 6]
```diff
@@ -18,7 +18,7 @@ export interface ManagerAttributes {
 
     user?: AppUser;
     community?: CommunityAttributes;
-    addedBeneficiaries?: UbiBeneficiaryRegistry[];
+    added?: UbiBeneficiaryRegistry[];
```
Comment (obernardovieira) [inline packages/core/src/database/models/ubi/manager.ts:21]: is this used anywhere?
if not, remove

**Review by obernardovieira — APPROVED** [review]
Just one little thing.
Everything else looks good.
Always try to query only what's really needed.

File: packages/core/src/database/models/ubi/manager.ts (hunk 6 above)
Comment (dev-jotape) [reply packages/core/src/database/models/ubi/manager.ts]: is not being used. 
removed.

**Pull request merged**